bugsnag-android-core/src/test/java/com/bugsnag/android/DocumentPathTest.kt

```diff
@@ -14,7 +14,10 @@ class DocumentPathTest {
         val expected = mapOf<String, Any>("a" to 1)
 
         val observed = docpath.modifyDocument(document, value)
-        Assert.assertEquals(expected, observed)
+        Assert.assertEquals(
+            BugsnagTestUtils.normalized(expected),
+            BugsnagTestUtils.normalized(observed)
+        )
```

Comment:
If several assertions require the exact same setup, some variety of custom assertion seems worthwhile to keep things concise. Maybe something like `assertEqualNormalized()`?

Reply:
I've moved it to https://github.com/bugsnag/bugsnag-android/pull/1332/files#diff-972efa5cecbfa9d46ef7af453c6890918e814547ff95356ce929762014f4f83cR206